packages/appstore/src/modules/onboarding/onboarding.tsx

```diff
@@ -72,6 +91,12 @@ const Onboarding = observer(({ contents = getTradingHubContents() }: TOnboarding
 
     const footer_description = is_eu_user ? eu_footer_text : footer_text;
 
+    useEffect(() => {
+        if (is_logged_in && is_landing_company_loaded) {
+            trackOnboardingOpen();
+        }
+    }, [is_logged_in, is_landing_company_loaded, is_first_time_visit, loginid, trackOnboardingOpen]);
```

Comment:
@amir-deriv are you sure you want to call `trackOnboardingOpen` every time one of these deps change?

Reply:
@maryia-deriv I have updated the deps now, I was using it for initial implementation. but now it's sorted